src/Analyzers/CSharp/Analyzers/RemoveUnreachableCode/RemoveUnreachableCodeHelpers.cs

```diff
@@ -12,15 +14,35 @@ internal static class RemoveUnreachableCodeHelpers
     {
         public static ImmutableArray<ImmutableArray<StatementSyntax>> GetSubsequentUnreachableSections(StatementSyntax firstUnreachableStatement)
         {
-            SyntaxList<StatementSyntax> siblingStatements;
+            StatementSyntax[] siblingStatements;
```

Comment:
I couldn't rely on `SyntaxList<TNode>` for global statements due to https://github.com/dotnet/roslyn/issues/61836.

Reply:
this would be worth putting a comment.  also, if you don't intend on mutating this, make into an immutablearray.